components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -167,34 +167,33 @@ public function options( $options, $pod ) {
 		// check if it's a taxonomy Pod, if so add fields for that
 		if ( $pod['type'] === 'taxonomy' ) {
 			$options['pods-pfat'] = array(
-				'pfat_enable'           => array(
-					'label'             => __( 'Enable Automatic Pods Templates for this Pod?', 'pods' ),
-					'help'              => __( 'When enabled you can specify the names of a Pods Template to be used to display items in this Pod in the front-end.', 'pods' ),
+				'pfat_enable'        => array(
+					'label'             => __( 'Enable Automatic Pods Templates for this Taxonomy Pod?', 'pods' ),
+					'help'              => __( 'When enabled you can specify the names of a Pods Template to be used to display information about this taxonomy and/or posts in this taxonomy in the front-end.', 'pods' ),
 					'type'              => 'boolean',
 					'default'           => false,
 					'dependency'        => true,
 					'boolean_yes_label' => '',
 				),
-				'pfat_run_outside_loop' => array(
-					'label'             => __( 'Execute Auto Template outside of the WordPress loop? (advanced)', 'pods' ),
-					'help'              => __( 'When enabled, the template will be executed whenever the specified filter is called.', 'pods' ),
-					'type'              => 'boolean',
-					'default'           => false,
-					'depends-on'        => array( 'pfat_enable' => true ),
-					'boolean_yes_label' => '',
-				),
-				'pfat_archive'          => array(
```

Comment:
Breaking compat possibly?

Reply:
I can't think of a case where this would happen.
Before this PR taxonomies didn't support the auto-templates anyway. The options were redundant...
@jimtrue Your thoughts?

Reply:
Closing this as resolved. Before this PR Archives didn't support auto templates so these options didn't have a purpose at all.